### PR TITLE
Last minute bee stuff

### PR DESCRIPTION
**Pull request by Shnupbups**
Mapped some unmapped things related to bees, changed `BeeHive` to `Beehive` to reflect it being one word in game now, other small changes

### CODE REVIEW

File: mappings/net/minecraft/block/BeeHiveBlock.mapping [hunk 1]
```diff
@@ -1,9 +1,33 @@
-CLASS net/minecraft/class_4481 net/minecraft/block/BeeHiveBlock
+CLASS net/minecraft/class_4481 net/minecraft/block/BeehiveBlock
```
Comment (Juuxel) [inline mappings/net/minecraft/block/BeeHiveBlock.mapping]: The file is still called `BeeHiveBlock.mapping` with a capital H.

File: mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping [hunk 2]
```diff
@@ -1,20 +1,34 @@
-CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeeHiveBlockEntity
+CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeehiveBlockEntity
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping:1]: Same here.

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 3]
```diff
@@ -20,6 +20,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 	METHOD method_11005 createFromTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2586;
 	METHOD method_11006 getSquaredRenderDistance ()D
 	METHOD method_11007 toTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487;
+		ARG 1 compoundTag
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping]:
```suggestion
		ARG 1 tag
```

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 4]
```diff
@@ -30,6 +31,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 	METHOD method_11012 markRemoved ()V
 	METHOD method_11013 applyRotation (Lnet/minecraft/class_2470;)V
 	METHOD method_11014 fromTag (Lnet/minecraft/class_2487;)V
+		ARG 1 compoundTag
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping]:
```suggestion
		ARG 1 tag
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/BeeHiveBlock.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/block/BeeHiveBlock.mapping]: Grr, blame Windows for that one. It's renamed on my actual computer, but Windows always acts weird with case in file names, so it didn't commit the name change.

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**